src/main/kotlin/uk/gov/gdx/datashare/service/EventDataService.kt

```diff
@@ -161,11 +169,44 @@ data class EventNotification(
   @Schema(description = "ID from the source of the notification", required = true, example = "999999901")
   val sourceId: String,
   @Schema(
-    description = "Event Data - only returned when the consumer has `enrichmentFieldsIncludedInPoll` enabled, otherwise an empty object. " +
-      "Full dataset for the event can be obtained by calling /events/{id}",
+    description = "Indicates that event data is returned when true,",
+    required = false,
+    example = "false",
+  )
+  val dataIncluded: Boolean? = null,
+  @Schema(
+    description = "list of data fields that will be returned in this event",
+    required = false,
+    example = "firstNames,lastName,sex,dateOfDeath",
+  )
+  val enrichmentFields: String? = null,
+  @Schema(
+    description = "<h2>Event Data</h2>" +
```

Comment:
I'd be keen for us to link out to the HMPO docs on this too to say "the authoritative source for this isn't us"

Reply:
Good idea